app/Http/Controllers/AdminDash.php

```diff
@@ -995,7 +996,7 @@ public function hideFeedback(Request $request, $id) {
 
     public function updateFeedback(Request $request, $id) {
         $feedback = Feedback::find($id);
-        $feedback->controller_id = $request->controller_id;
+        $feedback->controller_id = $request->feedback_id;
```

Comment:
Is there any way to get it to where we can set the feedback ID alias using `$feedback->feedback_id = ...` instead of controller_id?

Reply:
I'm not sure how to build an alias like that since controller_id is the table column name. I guess that would be the nuclear option... we could just manually change the column name.

Reply:
Yes, actually we can! I added a setAttribute method to the model and aliased the name to accomplish this.

Reply:
Sweet!